source/_components/xs1.markdown

```diff
@@ -0,0 +1,103 @@
+---
+layout: page
+title: "EZcontrol XS1"
+description: "Instructions on how to integrate an XS1 Gateway within Home Assistant."
+date: 2018-12-04 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xs1.jpg
+ha_category: Hub
+featured: true
+ha_iot_class: "Local Polling"
+---
```

Comment:
Please add `ha_release`.

Reply:
If there's not much info in the platform pages, we now prefer to not create those pages but just create artifical entries in the component index by adding categories and redirects here in the front matter.

See eg:
https://github.com/home-assistant/home-assistant.io/pull/8427

Reply:
ok